src/main/java/com/minecolonies/coremod/inventory/GuiItemHandler.java

```diff
@@ -0,0 +1,49 @@
+package com.minecolonies.coremod.inventory;
+
+import net.minecraft.client.Minecraft;
+import net.minecraft.client.gui.inventory.GuiContainer;
+import net.minecraft.client.renderer.GlStateManager;
+import net.minecraft.util.ResourceLocation;
+import net.minecraft.util.text.TextComponentTranslation;
+import net.minecraftforge.fml.relauncher.Side;
+import net.minecraftforge.fml.relauncher.SideOnly;
+
+import java.awt.*;
+
+@SideOnly(Side.CLIENT)
+public class GuiItemHandler extends GuiContainer
+{
+    private static final ResourceLocation TEXTURE = new ResourceLocation("textures/gui/container/generic_54.png");
+    private final IInteractiveItemHandler handler;
+    private final int inventoryRows;
+
+    public GuiItemHandler(IInteractiveItemHandler handler)
+    {
+        super(new ContainerItemHandler(Minecraft.getMinecraft().thePlayer.inventory, handler));
+        this.handler = handler;
+        this.allowUserInput = false;
+        this.inventoryRows = handler.getSlots() / 9;
+        this.ySize = 114 + this.inventoryRows * 18;
+    }
+
+    /**
+     * Draw the foreground layer for the GuiContainer (everything in front of the items)
+     */
+    protected void drawGuiContainerForegroundLayer(int mouseX, int mouseY)
+    {
+        this.fontRendererObj.drawString(new TextComponentTranslation(this.handler.getName()).getUnformattedText(), 8, 6, 4210752);
+    }
+
+    /**
+     * Draws the background layer of this container (behind the items).
+     */
+    protected void drawGuiContainerBackgroundLayer(float partialTicks, int mouseX, int mouseY)
+    {
+        GlStateManager.color(Color.WHITE.getRed(), Color.WHITE.getGreen(), Color.WHITE.getBlue(), Color.WHITE.getAlpha());
+        this.mc.getTextureManager().bindTexture(TEXTURE);
+        int i = (this.width - this.xSize) / 2;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Local variable 'i' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)